package.json

```diff
@@ -167,6 +167,7 @@
     "zurvan": "^0.5.2"
   },
   "scripts": {
+    
```

Comment:
can you remove this stray blank line?

Reply:
I removed this line also run the yarn lint --fix in ui.